Tools/clang-check.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
@mcharleb That sounds good. So in this case, Firmware/Tools/rpi/clang-check.sh?
@julianoes Are you happy with that too?